runtime-files: Update Headers

chrisbra opened the pull request:
After merging in commits 81b8bf5b4a33 and 6a500661a9cb, updating the header was forgotten.  So do it now.  While at it, also change the maintainer for the 'defaults.vim' runtime file from Bram to the Vim project.

k-takata commented on runtime/defaults.vim:
```diff
@@ -1,7 +1,8 @@
 "
-" Maintainer:	Bram Moolenaar <Bram@vim.org>
-" Last change:	2023 May 10
+" Maintainer:	The Vim Project https://github.com/vim/vim
```
```suggestion
" Maintainer:	The Vim Project <https://github.com/vim/vim>
```
Is it better to add `< >`?

chrisbra replied:
yes better. But let me remove defaults.vim here. It is handled as part of https://github.com/vim/vim/pull/12749

chrisbra committed:
runtime-files: Update Headers

After merging in commits 81b8bf5b4a33 and 6a500661a9cb, updating the
header was forgotten.  So do it now.
